docs/source/ngv.rst

```diff
@@ -15,6 +15,128 @@ The NGV simulation requires three node populations, each stored in separate file
+                "neurolucida-asc": "$BASE_DIR/morphologies/neurons"
+              }
+            }
+          }
+        },
+        {
+          "nodes_file": "$BASE_DIR/sonata/networks/nodes/astrocytes/nodes.h5",
+          "populations": {
+            "astrocytes": {
+              "type": "astrocyte",
+              "alternate_morphologies": {
+                "h5v1": "$BASE_DIR/morphologies/astrocytes/h5"
+              },
+              "microdomains_file": "$BASE_DIR/sonata/networks/nodes/astrocytes/microdomains.h5",
+              "provenance": {
+                "bioname_dir": "$BASE_DIR/../bioname"
+              }
+            }
+          }
+        },
+        {
+          "nodes_file": "$BASE_DIR/sonata/networks/nodes/vasculature/nodes.h5",
+          "populations": {
+            "vasculature": {
+              "type": "vasculature",
+              "vasculature_file": "$BASE_DIR/../atlas/vasculature.h5",
+              "vasculature_mesh": "$BASE_DIR/../atlas/vasculature.obj",
+              "provenance": {
+                "bioname_dir": "$BASE_DIR/../bioname"
+              }
+            }
+          }
+        }
+      ],
+      "edges": [
+        {
+          "edges_file": "$BASE_DIR/sonata/networks/edges/functional/neocortex_neurons__chemical_synapse/edges.h5",
+          "populations": {
+            "neocortex_neurons__chemical_synapse": {
+              "type": "chemical",
+              "provenance": {
+                "bioname_dir": "$BASE_DIR/../bioname"
+              },
+              "spatial_synapse_index_dir": "$BASE_DIR/sonata/networks/edges/functional/neocortex_neurons__chemical_synapse/spatial_synapse_index"
+            }
+          }
+        },
+        {
+          "edges_file": "$BASE_DIR/sonata/networks/edges/neuroglial/edges.h5",
+          "populations": {
+            "neuroglial": {
+              "type": "synapse_astrocyte",
+              "provenance": {
+                "bioname_dir": "$BASE_DIR/../bioname"
+              }
+            }
+          }
+        },
+        {
+          "edges_file": "$BASE_DIR/sonata/networks/edges/glialglial/edges.h5",
+          "populations": {
+            "glialglial": {
+              "type": "glialglial",
+              "provenance": {
+                "bioname_dir": "$BASE_DIR/../bioname"
+              }
+            }
+          }
+        },
+        {
+          "edges_file": "$BASE_DIR/sonata/networks/edges/gliovascular/edges.h5",
+          "populations": {
+            "gliovascular": {
+              "type": "endfoot",
+              "endfeet_meshes_file": "$BASE_DIR/sonata/networks/edges/gliovascular/endfeet_meshes.h5",
+              "provenance": {
+                "bioname_dir": "$BASE_DIR/../bioname"
+              }
+            }
+          }
+        }
+      ]
+    }
+  }
+
+The rest of the simulation can be found `here <https://github.com/BlueBrain/MultiscaleRun/tree/main/multiscale_run/templates/tiny_CI>`_.
```

Comment:
This big block from the config seems much, but maybe you intend to later break up the sections in order to give more explanatory text? Otherwise, it seems that it is sufficient to give the link?